index.css

```diff
@@ -152,6 +163,207 @@ li {
   align-items: center;
 }
 
+.textside p {
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills li {
+  background-color: #ebebff;
+  padding: 6px 5px;
+  border-radius: 8px;
+  font-size: 15px;
+  color: #396df2;
+  font-weight: 600;
+  line-height: 16px;
+}
+
+.about-skills li {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  padding: 20px;
+  background: #f7f7f9;
+  border-radius: 8px;
+  width: 140px;
+}
+
+.card-ul li {
+  font-weight: 400;
+  font-size: 18px;
+  line-height: 24px;
+  color: #6b778c;
+}
+
+.card-ul li:nth-child(1) {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.skills {
+  text-align: center;
+  display: flex;
+}
+
+.see-project {
+  background-color: #fff;
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 9px;
+  padding: 5px 8px;
+  align-self: flex-start;
+  letter-spacing: 0.03em;
+  width: 140px;
+  height: 50px;
+  font-size: 15px;
+  font-weight: 700;
+}
+
+.see-project:hover {
+  background: #6070ff;
+  box-shadow: #4053fc3d;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.see-project:active {
+  background: #2230d2;
+  border-radius: 8px;
+  color: #fff;
+}
+
+.see-project:disabled {
+  color: #5e6c84;
+  background-color: #fff;
+}
+
+.about-social {
+  gap: 10px;
+}
+
+#about {
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  background-color: #fff;
+  padding: 2em 2em;
+  gap: 10px;
+  border-radius: 20px;
+}
```

Comment:
- Great job so far @joeveloper but as the previous reviewer mentioned,
- [ ] This section consists of two parts - About Myself part and Language part. The about myself part takes up 40% of the width while the language part takes up the remaining 60% according to how we have it on the Figma design. 
- [ ] From your styles here, since you applied `grid-template-columns: repeat(2, 1fr)`, Both of them will have the same size. So I will suggest that in order to meet the requirement, you use `(grid-template-columns: 40% 60%)`. :100: 
- [ ]  Also, just as the previous reviewer mentioned, please adjust the vertical margin among `LET'S CONNECT`, `social icons`, and the `Get My Resume` button. :+1: 
- Your Design
![image](https://user-images.githubusercontent.com/102699958/220724556-5fbbf14a-3b09-48d3-9981-33c88b91b242.png)

- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/220724693-334251c3-6497-474c-981e-d4609deba49a.png)

Reply:
gap has been resolved appropriately